projects/Mallard/src/authentication/auth-context.tsx

```diff
@@ -189,6 +190,16 @@ const AuthProvider = ({ children }: { children: React.ReactNode }) => {
                                 'live',
                             ),
                         )
+                    } else {
+                        Alert.alert(
```

Comment:
You might want to check with Design, but I'm pretty sure there's a design for this.